pyuvdata/uvdata/tests/test_mir.py

```diff
@@ -124,7 +148,9 @@ def test_write_mir(uv_in_uvfits, err_type=NotImplementedError):
         mir_uv.write_mir("dummy.mir")
 
 
-def test_flex_spw_read(tmp_path):
+@pytest.mark.filterwarnings("ignore:LST values stored in this file are not ")
+@pytest.mark.parametrize("future_shapes", [True, False])
+def test_flex_spw_read(tmp_path, future_shapes):
```

Comment:
I wonder whether this test should be moved to `test_uvdata` because it is effectively designed to test UVData functionality, it just happens to use a MIR file because it's a file with flexible spectral windows. It probably doesn't matter too much, but if I was looking for the test that tested this functionality I wouldn't look in this file right away.

Reply:
I'd certainly be amenable to moving it over to `test_uvdata` -- it started in `test_mir` because that just so happened to be the first file format that supported it in early testing, but it doesn't need to stay that way.